noir-projects/aztec-nr/aztec/src/note/retrieved_note.nr

```diff
@@ -10,6 +10,7 @@ pub struct RetrievedNote<Note> {
     pub contract_address: AztecAddress,
     pub owner: AztecAddress,
     pub randomness: Field,
+    pub storage_slot: Field,
```

Comment:
These docs now need updating - this is now the information that is needed to prove a note exists.